internal/contributebot/process.go

```diff
@@ -161,6 +161,13 @@ func processPullRequestEvent(data *pullRequestData) *pullRequestEdits {
 	defer log.Printf("-> %v", edits)
 	pr := data.PullRequest
 
+	// Skip the process when the PR is closed, we check this here instead of when
```

Comment:
Can you add a similar check in processIssueEvent? It will have to be after the in-progress label check, since that is explicitly for closed issues. The rest of the checks in there should be skipped for closed issues though.

Reply:
Done.